src/pages/Environments.js

```diff
@@ -196,7 +195,7 @@ const Environments = () => {
   const totalRuntimeCost = _.sum(_.map(runtimeCost, runtimes))
   const totalAppCost = _.sum(_.map(getGalaxyComputeCost, apps))
   const totalCost = totalRuntimeCost + totalAppCost
-  const totalDiskCost = _.sum(_.map(getPersistentDiskCostMonthly, disks))
```

Comment:
We were not passing in the second (`region`) parameter into `getPersistentDiskCostMonthly` function.